README.md

```diff
@@ -40,93 +40,72 @@
 
+Planned features:
+- Display options for execution data with interactive capacities.
+- BPMN extensions
+- Library extension points
 
 ## 🌏 Browser Support
 
+**We do our best to support recent versions of major browsers**
+
 | <img src="https://www.google.com/chrome/static/images/chrome-logo.svg" alt="Chrome" width="18px" height="18px" /> Chrome | <img src="https://user-media-prod-cdn.itsre-sumo.mozilla.net/uploads/products/2020-04-14-08-36-13-8dda6f.png" alt="Firefox" width="18px" height="18px" /> Firefox | <img src="https://developer.apple.com/assets/elements/icons/safari/safari-96x96.png" alt="Safari" width="18px" height="18px" /> Safari | <img src="https://avatars0.githubusercontent.com/u/11354582?s=200&v=4" alt="Edge" width="18px" height="18px" /> Edge |
 | :---------: | :---------: | :---------: | :---------: |
 |  ✔️ |  ✔️ |  ✔️ |  ✔️ |
 
 **Note**: Internet Explorer will never be supported. \
 The library may work with the other browsers. They must at least support ES6.
 
-## 🎨 Features
-
-`bpmn-visualization` is in early development stages and is subject to changes prior to the `1.0.0` release.\
-\
-Implemented features:
-- Rendering of the most common BPMN elements.  
-Notice that there is currently no plan to support `Conversation` and `Choreography`.
-
-Current implementation:
-- The [packaging](https://github.com/process-analytics/bpmn-visualization-js/milestone/18) of the library.
-
-Future features:
-- BPMN extensions
-- The library extension points
-- Display options for execution data with interactive capacities.
-
 
 ## ♻️ Usage
-We support various module formats on [NPM](https://npmjs.org/package/bpmn-visualization):
-- [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE)
-- [ESM](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm)
-- [CommonJS](https://www.typescriptlang.org/docs/handbook/2/modules.html#commonjs)
+Library is available through [NPM](https://npmjs.org/package/bpmn-visualization) \
+We support various module formats such as:
+[IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE),
+[ESM](https://dev.to/iggredible/what-the-heck-are-cjs-amd-umd-and-esm-ikm),
+[CommonJS](https://www.typescriptlang.org/docs/handbook/2/modules.html#commonjs).
   
-
-### IIFE
-See how it's done in [this example](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/examples/01-getting-started).
-
-
-### ESM
-🚀 You can find an integration example of Vanilla Typescript with rollup, [here](https://github.com/process-analytics/bpmn-visualization-examples/tree/master/examples/projects/typescript-vanilla-with-rollup).
+Here are examples per module: \
+[bpmn-visualization (IIFE)](https://github.com/process-analytics/bpmn-visualization-examples/blob/master/examples/01-getting-started) \
+[bpmn-visualization (ESM); Typescript + Rollup](https://github.com/process-analytics/bpmn-visualization-examples/tree/master/examples/projects/typescript-vanilla-with-rollup)
```

Comment:
I would remove it, why not providing few commands a most project does?
Further details are available in the examples.
We won't be able to put enough information in the README on that topic, so I suggest to drop everything instead of just putting information that are not enough for a whole understanding and the whole usage